subsys/nrf_security/src/drivers/cracen/cracenpsa/src/common.c

```diff
@@ -23,21 +23,13 @@
 #include <sxsymcrypt/sha2.h>
 #include <sxsymcrypt/sha3.h>
 #include <zephyr/sys/util.h>
+#include <psa/nrf_platform_key_ids.h>
 
 #define NOT_ENABLED_CURVE    (0)
 #define NOT_ENABLED_HASH_ALG (0)
 
 #ifdef NRF54H_SERIES
-/* NCSDK-27273: These defines will come from an external header file. */
-#define DOMAIN_NONE	   0x00
-#define DOMAIN_SECURE	   0x01
-#define DOMAIN_APPLICATION 0x02
-#define DOMAIN_RADIO	   0x03
-#define DOMAIN_CELL	   0x04
-#define DOMAIN_ISIM	   0x05
-#define DOMAIN_WIFI	   0x06
-#define DOMAIN_SYSCTRL	   0x08
-
+/* NCSDK-27273: These defines will come from MDK */
```

Comment:
Nit but the comment is a bit off right? If the defines that you are talking about is the device secret length I guess it is not relevant to the jira ticket.